include/openssl/opensslv.h.in

```diff
@@ -36,55 +36,24 @@ extern "C" {
 # define OPENSSL_VERSION_PATCH  {- $config{patch} -}
 
 /*
- * Additional version information, defined only when used.
+ * Additional version information
  *
  * These are also part of the new version scheme, but aren't part
  * of the version number itself.
  */
 
 /* Could be: #define OPENSSL_VERSION_PRE_RELEASE "-alpha.1" */
-{- $config{prerelease}
-    ? << "_____"
-# define OPENSSL_VERSION_PRE_RELEASE "$config{prerelease}"
-_____
-    : << "_____"
-# undef OPENSSL_VERSION_PRE_RELEASE
-_____
--}
+# define OPENSSL_VERSION_PRE_RELEASE "{- $config{prerelease} -}"
 /* Could be: #define OPENSSL_VERSION_BUILD_METADATA "+fips" */
 /* Could be: #define OPENSSL_VERSION_BUILD_METADATA "+vendor.1" */
-{- $build_metadata
-    ? << "_____"
 # define OPENSSL_VERSION_BUILD_METADATA "{- $config{build_metadata} -}"
-_____
-    : << "_____"
-# undef OPENSSL_VERSION_BUILD_METADATA
-_____
--}
 
 /*
- * Note: OPENSSL_VERSION_BUILD_METADATA will never be defined by
- * the OpenSSL Project, it's entirely reserved for others vendors
+ * Note: The OpenSSL Project will never define OPENSSL_VERSION_BUILD_METADATA
+ * to be anything but the empty string.  Its use is entirely reserved for
+ * others vendors
```

Comment:
others->other  or strike the word vendors.

Reply:
Fixed